src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/ComInterfaceGenerator.cs

```diff
@@ -393,6 +393,46 @@ private static IncrementalMethodStubGenerationContext CalculateStubInformation(M
             // Create the stub.
             var signatureContext = SignatureContext.Create(symbol, DefaultMarshallingInfoParser.Create(environment, generatorDiagnostics, symbol, new InteropAttributeCompilationData(), generatedComAttribute), environment, typeof(VtableIndexStubGenerator).Assembly);
 
+            for (var i = 0; i < signatureContext.ElementTypeInformation.Length; ++i)
```

Comment:
I don't know what this loop is doing. It is not simple and has a lot of context. Let's add a comment in the same way we do above for things.